torch/csrc/jit/operator.cpp

```diff
@@ -114,6 +114,17 @@ struct OperatorRegistry {
     }
     return ret;
   }
+
+  const std::vector<std::shared_ptr<Operator>>& getAllOperators() {
+    std::lock_guard<std::mutex> guard(lock);
+    registerPendingOperators();
+    static std::vector<std::shared_ptr<Operator>> values;
```

Comment:
returning pointer to a static is not a good idea (as it can theoretically race. Since perf doesn't matter - just allocate new vector eac time

Reply:
interesting... I thought with lock_guard, race should be gone.

Anyway, changed to allocating a new vector.

Reply:
you return the pointer to `values`, someone can read it when `guard` is no longer held (but instead if taken by another thread that repopulates `values` from scratch)